packages/react-composites/src/composites/ChatComposite/ChatComposite.tsx

```diff
@@ -47,10 +55,18 @@ export type ChatOptions = {
 };
 
 export const ChatComposite = (props: ChatCompositeProps): JSX.Element => {
-  const { adapter, fluentTheme, options, identifiers, onRenderAvatar, onRenderTypingIndicator, onRenderMessage } =
-    props;
+  const {
```

Comment:
I'm personally not at all a fan of destructuring props into local variables.

I'd much rather see `props.X` to know what comes from the props vs what is a local variable...

Reply:
This is a tangential comment... Feel free to ignore for this PR :D